source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -48,6 +47,10 @@ namespace AdaptiveCards
                 Windows.UI.Xaml.ResourceDictionary OverrideStyles;
                 AdaptiveFeatureRegistration FeatureRegistration;
 
+                Boolean OverflowMaxActions;
```

Comment:
Is the overflow of actions only supposed to happen on the card level ActionSet (the counterpart to the card body)? Will other ActionSet elements inherit the same overflow text and accessibility texts?

Reply:
Overflow happens on all action sets and they will all use this same set of properties.